### DISCUSSION TITLE
Arrange a meeting for those interested in developing feature file formats

### DISCUSSION THREAD

**Discussion by simoncozens** [adobe-type-tools/afdko#1708] [General]
As discussed in the UFO meeting.

**Comment by simoncozens** [comment]
@tamirhassan

**Comment by josh-hadley** [comment]
@simoncozens from the UFO meeting way back when, our understanding was that you were going to organize this meeting. Are you still planning to do that?

**Comment by simoncozens** [comment]
I am. Sorry, I dropped the ball here. Today I was reminded, and Tamir wants to attend. I'll propose some dates.

**Comment by simoncozens** [comment]
This Doodle poll has a range of times (afternoon/evening Europe, morning/daytime US) for the week after next:

https://doodle.com/poll/g2frbryziag32qqh

I'll put together a suggested agenda on Monday.

**Comment by simoncozens** [comment]
@punchcutter @cjchapman @alerque @typemytype @mathieureguer @arialcrime @typoman @mhosken.

**Comment by cjchapman** [comment]
@schriftgestalt, @frankrolf, & @kaydeearts see the Doodle poll above about setting a time for "a meeting for those interested in developing feature file formats".

**Comment by schriftgestalt** [comment]
Thanks for mentioning this.

**Comment by lianghai** [comment]
@simoncozens Seems like Monday evening is preferred by the most? If we start at 5 pm **[UTC+1—afterthought]** we get Cosimo.

**Comment by tamirhassan** [comment]
@simoncozens Is the meeting time decided yet? If it's going to be on the coming Monday, it would be great to know the time a little in advance. Thanks!

**Comment by simoncozens** [comment]
Sorry, yes - this coming Monday, 4pmUTC! I think I actually intended a week later so I thought I still had lots of time.

Zoom invitation below. Waiting room enabled. For security if I don’t recognise your name, bear with me and I’ll ask you to ping me via another medium.

Rough agenda:

1) *Brief* intro to existing projects - 3 mins per project max.
2) Approaches to “variable layout” - 20 mins max
3) Notable features from existing projects and desires for other people’s projects! - remainder of time.

Simon Cozens is inviting you to a scheduled Zoom meeting.

Topic: Feature File Format Chat
Time: Mar 15, 2021 04:00 PM London

Join Zoom Meeting
https://us02web.zoom.us/j/83503692306?pwd=T0R1TGtRNG5GY1ZlUWx1VEdYcHR3dz09

Meeting ID: 835 0369 2306
Passcode: 725926
One tap mobile
+12532158782,,83503692306#,,,,*725926# US (Tacoma)
+13017158592,,83503692306#,,,,*725926# US (Washington DC)

Dial by your location
        +1 253 215 8782 US (Tacoma)
        +1 301 715 8592 US (Washington DC)
        +1 312 626 6799 US (Chicago)
        +1 346 248 7799 US (Houston)
        +1 646 558 8656 US (New York)
        +1 669 900 9128 US (San Jose)
Meeting ID: 835 0369 2306
Passcode: 725926
Find your local number: https://us02web.zoom.us/u/kARMb70li

**Comment by simoncozens** [comment]
Apologies : 4pm UTC, not 5. (I blame liang but really it’s just that I’m terrible at anything administrative...) Post and invite updated.

**Comment by lianghai** [comment]
> Apologies : 4pm UTC, not 5. (I blame liang but really it’s just that I’m terrible at anything administrative...) Post and invite updated.

(Hahaha my bad… Somehow I entirely forgot about the timezone issue—and lived in a perfect world for a minute.)

**Comment by tiroj** [comment]
Phooey. I have another meeting at the same time.

If anyone feels like playing my role in this meeting: just repeatedly state that the feature file format should be suitable to live at a level below graphical user interfaces, higher-level input and programming languages, and generally be something that font makers should only need to look at as a last resort.

**Comment by schriftgestalt** [comment]
We wrote down some additions we did/are planing to the fea syntax in Glyphs:
https://github.com/schriftgestalt/GlyphsSDK/wiki/Extending-the-Feature-File-Syntax

**Comment by mhosken** [comment]
What descriptive elements would  you see as needing to be accessed from a
GUI? I know of APs and the insertion of tagged code from code generators.
Anything else?

**Comment by simoncozens** [comment]
I've put together a slide deck for later today, which summarises what I know of existing projects/suggestions. Please feel free to comment:

https://docs.google.com/presentation/d/1hcfx2kE8GBQ5bpj67GyE2oQ0ufy-I0JlmWg5czqZocw/edit?usp=sharing

**Comment by frankrolf** [comment]
Just in case … https://www.worldtimebuddy.com/

**Comment by belluzj** [comment]
Thanks for organizing! Will there be meeting minutes? Do you need help with taking notes? (I will attend unless a previous meeting overruns).

**Comment by simoncozens** [comment]
We'll take minutes in a Google doc, yes.

**Comment by simoncozens** [comment]
Actually I will probably need help taking notes because I don't think I'll be able to share slides and take notes at the same time.

**Comment by simoncozens** [comment]
If you’re “RL” and you’re trying to get into the meeting, please let me know who you are.

**Comment by belluzj** [comment]
Where are you taking notes?

**Comment by bobh0303** [comment]
Notes: https://docs.google.com/document/d/1kZmZWO_dH8H91Kue0r2-m7ob9X1y31W-KdpHf23XpLw/edit
